lms/static/sass/course/wiki/_basic-html.scss

```diff
@@ -80,58 +86,108 @@ section.wiki-body {
       font-size: 1.1em;
     }
 
-    h4, p, blockquote, ul, fieldset, form, ol, dl, dir, menu  { margin: 1.12em 0 }
+    h4, p, blockquote, ul, fieldset, form, ol, dl, dir, menu  {
+      margin: 1.12em 0;
+    }
 
-    h5 { font-size: 1em; margin: 1.5em 0 }
+    h5 {
+      font-size: 1em;
+      margin: 1.5em 0;
+    }
 
-    h6 { font-size: 0.5em; margin: 1.67em 0 }
+    h6 {
+      font-size: 0.5em;
+      margin: 1.67em 0;
+    }
 
-    b, strong { font-weight: bolder }
+    b, strong {
+      font-weight: bolder;
+    }
 
     blockquote      {
```

Comment:
I change all opening braces should be preceded by a single space, its a good for readability.
@alisan617 what you think?